configure.ac

```diff
@@ -94,7 +94,19 @@ AC_ARG_ENABLE(multilib,
 	[enable_multilib=no]
 	)
 
-AS_IF([test "x$enable_multilib" != xno],
+AC_ARG_WITH(multilib-generator,
+	[AS_HELP_STRING([--with-multilib-generator],
+		[Multi-libs configuration string, only supported for bare-metal/elf toolchaih])],
```

Comment:
This implies --enable-multilib.  That should be documented here.

Reply:
Updated.